libvirt/tests/src/cpu/vcpu_misc.py

```
@@ -92,6 +92,10 @@ def update_cpu_xml(vmxml, params, test):
     with_topology = "yes" == params.get("with_topology", "no")
     customize_cpu_features = "yes" == params.get("customize_cpu_features", "no")
 
+    # Check for machine type for mode as maximum
+    if (platform.machine() in ['ppc64', 'ppc64le']) and (cpu_mode == "maximum"):
```

Ruff (0.14.5)

Undefined name `platform`

(F821)